x/tokenregistry/keeper/keeper.go

```diff
@@ -85,75 +85,67 @@ func (k keeper) CheckDenomPermissions(ctx sdk.Context, denom string, requiredPer
 	return true
 }
 
+func (k keeper) GetDenomPrefix(ctx sdk.Context, denom string) []byte {
```

Comment:
Does this need to be visible outside of the package?

Reply:
If not, I think we can make it lowercase

Reply:
should be lowercase, not used outside.